% Marieke Kuijjer
% 2013-02-15
% chapter 04

%	\documentclass[12pt,b5paper]{book}
%	\setcounter{secnumdepth}{0}
%	\setcounter{tocdepth}{1}
%	\usepackage[hidelinks]{hyperref}

%\begin{document}
%%%

%%% title page

\chapter{Tumor\hyp{}infiltrating macrophages are associated with metastasis suppression in high\hyp{}grade osteosarcoma: a rationale for treatment with macrophage activating agents}\label{ch4}
\thispagestyle{empty}				%%% to remove page number from first page of chapter, must be placed after calling the chapter

\vfill

\vspace{0.5cm}
This chapter is based on the publication:
Buddingh EP$\dagger$, \underline{Kuijjer ML}$\dagger$, Duim RA, B{\"u}rger H, Agelopoulos K, Myklebost O, Serra M, Mertens F, Hogendoorn PCW, Lankester AC, Cleton-Jansen AM. {\it Clin Cancer Res}. 2011 Apr 15;17(8):2110-9
\begin{small}
$\dagger$Shared first authorship
\end{small}

\newpage

%%% main document

\section{Abstract}\label{abstract4}
\textbf{Purpose}: High-grade osteosarcoma is a malignant primary bone tumor with a peak incidence in adolescence. Overall survival (OS) of patients with resectable metastatic disease is approximately 20\%. The exact mechanisms of development of metastases in osteosarcoma remain unclear. Most studies focus on tumor cells, but it is increasingly evident that stroma plays an important role in tumorigenesis and metastasis. We investigated the development of metastasis by studying tumor cells and their stromal context.

\textbf{Experimental Design}: To identify gene signatures playing a role in metastasis, we carried out genome\hyp{}wide gene expression profiling on prechemotherapy biopsies of patients who did ($n=34$) and patients who did not ($n=19$) develop metastases within 5 years. Immunohistochemistry (IHC) was performed on pretreatment biopsies from 2 additional cohorts ($n=63$ and $n=16$) and corresponding postchemotherapy resections and metastases.

\textbf{Results}: A total of 118/132 differentially expressed genes were upregulated in patients without metastases. Remarkably, almost half of these upregulated genes had immunological functions, particularly related to macrophages. Macrophage\hyp{}associated genes were expressed by infiltrating cells and not by osteosarcoma cells. Tumor\hyp{}associated macrophages (TAM) were quantified with IHC and associated with significantly better overall survival (OS) in the additional patient cohorts. Osteosarcoma samples contained both M1- (CD14/HLA-DR$\upalpha$ positive) and M2-type TAMs (CD14/CD163 positive and association with angiogenesis).

\textbf{Conclusions}: In contrast to most other tumor types, TAMs are associated with reduced metastasis and improved survival in high\hyp{}grade osteosarcoma. This study provides a biological rationale for the adjuvant treatment of high\hyp{}grade osteosarcoma patients with macrophage activating agents, such as muramyl tripeptide.

\section{Introduction}\label{introduction4}
High-grade osteosarcoma is a malignant bone tumor
characterized by the production of osteoid. The highest
incidence is in adolescent patients, with a second peak in
patients older than 40 years~\cite{raymond2002conventional}. Despite wide\hyp{}margin
surgery and intensification of chemotherapeutic treatment,
overall survival (OS) rates have reached a plateau at about
60\%~\cite{lewis2007improvement,bacci2006prognostic,bielack2002prognostic}. Novel administration modalities are needed,
but data on critical biological mechanisms allowing the
development of novel therapeutic agents are scarce for this
relatively rare tumor. In addition to conventional chemotherapeutic
agents, recent trials have explored immunostimulatory
strategies. The ongoing EURAMOS-1 trial
randomizes for treatment with IFN-$\upalpha$ in patients with good
histological response to neoadjuvant chemotherapy~\cite{marina2009international}. A
recently published clinical trial has shown improved OS for
osteosarcoma patients treated with the macrophage activating
agent muramyl tripeptide (MTP) added to the standard
chemotherapy regimen~\cite{meyers2008osteosarcoma}. However, only limited information
on macrophage infiltration and activation in osteosarcoma
is available~\cite{kleinerman1992unique}.

Tumor\hyp{}associated macrophages (TAM) may promote
tumorigenesis through immunosuppression, expression
of matrix\hyp{}degrading proteins and support of angiogenesis.
In numerous cancer types, high numbers of M2 or `alternatively
activated' TAMs are associated with a worse prognosis~\cite{hagemann2006ovarian,lee2008prognostic,lissbrant2000tumor,volodko1998tumour,jensen2009macrophage,van2009anti}. M2 macrophages have important functions
in wound healing and angiogenesis, express high levels of
the immunosuppressive cytokines interleukin (IL)-10 and
TGF-$\upbeta$, and express scavenger receptors such as CD163~\cite{sica2008macrophage,qian2010macrophage}.
`Classical activation' of macrophages by IFN-$\upgamma$ or
microbial products results in polarization toward M1-type
macrophages. M1 macrophages express high levels of
proinflammatory cytokines such as IL-12, IL-1, and IL-6
and have potent antitumor efficacy, both by reactive oxygen
species and cytokine\hyp{}induced cytotoxicity and by
induction of natural killer (NK) and T cell activity~\cite{mosser2008exploring}.
Rarely, high numbers of TAMs are associated with better
prognosis~\cite{kim2008high,forssell2007high}. In these cases, TAMs are presumably
polarized toward an M1 phenotype, although macrophage
subtypes were not reported in these two studies. Alternatively,
macrophages may directly phagocytose tumor cells, as has
been shown in acute myeloid leukemia~\cite{jaiswal2010macrophages}.

To investigate the role of stroma and stroma--tumor
interactions important in metastasis of osteosarcoma, we
investigated the development of metastasis by studying
tumor cells and their stromal context. By using genome\hyp{}wide
expression analysis, we showed that high expression
of macrophage\hyp{}associated genes in pretreatment biopsies
was associated with a lower risk of developing metastases.
In addition, we quantified and characterized TAMs in two
independent cohorts, including pretreatment biopsies,
postchemotherapy resections, and metastatic lesions. In
contrast to the tumor\hyp{}supporting role for TAMs in most
epithelial tumor types, higher numbers of infiltrating TAMs
correlated with better survival in osteosarcoma. Our findings
suggest that macrophages have direct or indirect antiosteosarcoma
activity and provide a possible explanation
for the beneficial effect of treatment with macrophage
activating agents in osteosarcoma.

\section{Materials and methods}\label{methods4}
\subsection{Patient cohorts}
Genome\hyp{}wide expression profiling was performed on
snap\hyp{}frozen pretreatment diagnostic biopsies containing
viable tumor material of 53 resectable high\hyp{}grade osteosarcoma
patients from the EuroBoNet consortium (\url{www.eurobonet.eu}; cohort 1). For immunohistochemical
validation, a tissue microarray containing 145 formalin\hyp{}fixed
paraffin\hyp{}embedded (FFPE) samples of 88 consecutive
high\hyp{}grade osteosarcoma patients with primary resectable
disease (cohort 2) and 29 FFPE samples of a cohort of 20
consecutive high\hyp{}grade osteosarcoma patients with resectable
disease were used (cohort 3), including material from
pretreatment biopsies, postchemotherapy resections, and
metastatic lesions~\cite{mohseny2009osteosarcoma}. Clinicopathological details can be
found in Supplemental Table S1 ({\it available online}~\cite{ch4additional}). All biological material
was handled in a coded fashion. Ethical guidelines of the
individual European partners were followed and samples
and clinical data were stored in the EuroBoNet biobank.

\subsection{Cell lines}
The 19 osteosarcoma cell lines 143B, HAL, HOS,
IOR/MOS, IOR/OS10, IOR/OS14, IOR/OS15, IOR/OS18,
IOR/OS9, IOR/SARG, KPD, MG-63, MHM, MNNG-HOS, OHS, OSA,
SAOS-2, U-2 OS, and ZK-58 were maintained in RPMI
1640 (Invitrogen) supplemented with 10\% fetal calf serum
and 1\% penicillin/streptomycin (Invitrogen) as previously
described~\cite{ottaviano2010molecular}.

\subsection{RNA isolation, cDNA synthesis, cRNA amplification,
and Illumina Human-6 v2.0 Expression BeadChip
hybridization}
Osteosarcoma tissue was snap\hyp{}frozen in 2-methylbutane
(Sigma\hyp{}Aldrich) and stored at 70$^\circ$C. By using a cryostat,
20mm sections from each block were cut and stained with
hematoxylin and eosin to ensure at least 70\% tumor
content and viability. RNA was isolated with TRIzol (Invitrogen),
followed by RNA cleanup using the QIAGEN
Rneasy mini kit with on\hyp{}column DNase treatment. RNA
quality and concentration were measured using an Agilent
2100 Bioanalyzer and Nanodrop ND-1000 (Thermo Fisher
Scientific), respectively. Synthesis of cDNA, cRNA amplification,
and hybridization of cRNA onto the Illumina
Human-6 v2.0 Expression BeadChips was carried out as
per manufacturer's instructions.

\subsection{Reverse transcriptase quantative PCR}
Reverse transcriptase quantative PCR (qPCR) analysis
of selected target genes was performed as previously
described~\cite{rozeman2005absence}. Each experiment was conducted in duplicate
by using an automated liquid\hyp{}handling system (Tecan,
Genesis RSP 100). Data were normalized by geometric
mean expression levels of 3 reference genes, {\it i.e.} {\it SRPR},
{\it CAPNS1}, and {\it TBP} using geNorm (\url{medgen.ugent.be/~jvdesomp/genorm/}). Primer sequences can be found in
Supplemental Table S2 ({\it available online}~\cite{ch4additional}).

\subsection{Enzymatic and fluorescent immunostainings}
Enzymatic and fluorescent immunostainings were performed
on 4mm sections of FFPE tissue as previously
described~\cite{mohseny2009osteosarcoma}. Details regarding antibodies and procedures
can be found in Supplemental Table S3 ({\it available online}~\cite{ch4additional}). In case of
double immunohistochemistry (IHC), incubation with
anti\hyp{}CD45 and development with DAB+ (Dako) occurred
first, followed by a second antigen retrieval before incubation
with either anti\hyp{}CD163 or anti\hyp{}HLA-DR$\upalpha$ and development
using the alkaline phosphatase substrate Vector
Blue (Vector Labs). In case of double immunofluorescent
(IF) stainings, primary antibodies were coincubated overnight.
As a positive control, normal and formic acid decalcified
tonsil was used, and as a negative control, no
primary antibody was added. Tissue microarray slides were
scanned using the MIRAX SCAN slide scanner and software
(Zeiss, Mirax 3D Histech). Numbers of positively stained
cells and vessels were counted using ImageJ (National
Institutes of Health, Bethesda, MD) and averaged per
0.6mm core. IF and double IHC images were acquired
using a Leica DM4000B microscope fitted with a CRI
Nuance spectral analyzer (Cambridge Research and Instrumentation,
Inc.) and analyzed using the supplied colocalization
tool to determine the percentage of single and
double positive pixels per region of interest.

\subsection{Microarray data analysis}
Gene expression data were exported from BeadStudio
version 3.1.3.0 (Illumina) in GeneSpring probe profile
format and processed and analyzed using the statistical
language R~\cite{r2.9.0}. As Illumina identifiers are not stable and
consistent between different chip versions, raw oligonucleotide
sequences were converted to nuIDs~\cite{du2007nuid}. Data
were transformed using the variance stabilizing transformation
algorithm to take advantage of the large number of
technical replicates available on the Illumina BeadChips~\cite{lin2008model}.
Transformed data were normalized using robust
spline normalization, an algorithm combining features
of quantile and loess normalization, specifically designed
to normalize variance\hyp{}stabilized data. All microarray data
processing was carried out by Bioconductor package {\it lumi}~\cite{gentleman2004bioconductor,du2008lumi}.
Quality control was performed using Bioconductor
package {\it arrayQualityMetrics}~\cite{kauffmann2009arrayqualitymetrics}. MIAME (minimum
information about a microarray experiment) compliant
data have been deposited in the GEO database (\url{www.ncbi.nlm.nih.gov/geo/}, accession number GSE21257).

\subsection{Statistical analysis}
Differential expression between patients who did ($n=34$) and did not ($n=19$) develop metastases within 5 years
from diagnosis of the primary tumor was determined using
linear models for microarray data ({\it LIMMA}~\cite{smyth2004linear}), applying
a Benjamini and Hochberg false discovery rate\hyp{}adjusted p-value cutoff of 0.05. Other univariate statistical
analyses were performed using GraphPad Prism Software
(version 5.01). Multivariate survival analyses were carried
out according to the Cox proportional hazards model in
SPSS (version 16.0.2). Two\hyp{}sided p-values $<0.05$ were determined to be significant; p-values between 0.05 and
0.15 were defined to be a trend.

\section{Results}\label{results4}
\subsection{High expression of macrophage\hyp{}associated genes in osteosarcoma biopsies of patients who did not develop metastases within 5 years from diagnosis (cohort 1)}
Comparison of genome\hyp{}wide gene expression in tumors
of patients who did and did not develop metastases within
5 years resulted in 139 significantly differentially expressed
(DE) probes, of which 125 corresponded to 118 upregulated
and 14 to downregulated genes in patients who did
not develop metastases. A summary of DE genes and
detailed descriptions of all probes can be found in Table~\ref{tab4.1}
and Supplemental Table S4 ({\it available online}~\cite{ch4additional}), respectively. Two DE genes
were specific for macrophages ({\it CD14} and {\it MSR1}) and 30/132 of the DE genes were associated with macrophage
functions such as antigen processing and presentation
({\it e.g.} {\it HLA-DRA} and {\it CD74}) or pattern recognition ({\it e.g.}
{\it TLR4} and {\it NLRP3}). Overall, approximately 20\% of the
upregulated probes corresponded to genes that were associated
with macrophage function and development and an
additional 25\% of the upregulated probes corresponded to
genes with other immunological functions, such as cytokine
production and phagocytosis. Four genes were
selected for validation of the microarray data using qPCR:
{\it CD14}, {\it HLA-DRA}, {\it CLEC5A}, and {\it FCGR2A}. Expression
levels as determined by qPCR correlated well with
expression levels obtained by microarray analysis (Supplemental
Figure S1 ({\it available online}~\cite{ch4additional})). Metastases\hyp{}free survival curves of the same
cohort, generated using median expression of the probe of
interest as a cutoff determining low and high expression,
are shown in Figure~\ref{fig4.1}B and Supplemental Figure 2 ({\it available online}~\cite{ch4additional}). Cox
proportional hazards analysis revealed expression of
macrophage\hyp{}associated genes {\it CD14} and {\it HLA-DRA} to be
independently associated with metastasis\hyp{}free survival
(Supplemental Table S5 ({\it available online}~\cite{ch4additional})).
%
\begin{landscape}
	\begin{table}[htbp]
		\centering
		\small
		\begin{tabular}{|>{\raggedright}p{2.3in} >{\raggedleft}p{0.6in} >{\raggedleft}p{0.6in} >{\raggedright}p{3in} >{\raggedleft}p{0.6in} >{\raggedleft}p{0.6in} l|}
			\hline
			& \multicolumn{3}{c}{Higher expression in patients without metastases} & \multicolumn{3}{l|}{Lower expression in these patients}\\
			\cmidrule(r){2-4}\cmidrule(r){5-7}
			\\[-3em]\mystrut
		& Number of probes & Number of genes & Examples & Number of probes & Number of genes & Examples\\
			\hline
			\rule{-4pt}{3ex} Pattern recognition receptor or signaling & 18 & 17 & {\it MSR1}, {\it CD14}, {\it NLRP3}, {\it TLR7}, {\it TLR8}, {\it TLR4}, {\it NAIP}, {\it IL1B}, {\it PYCARD}, {\it NLRC4} & 0 & 0 & \\
			Immunological & 16 & 15 & {\it CD86}, {\it C1QA}, {\it LY9}, {\it CD37}, {\it LY86} & 0 & 0 & \\
			HLA class II & 12 & 12 & {\it HLA-DMB}, {\it HLA-DRA}, {\it CD74}, {\it HLA-DQA1} & 0 & 0 & \\
			Hematopoietic cells & 11 & 10 & {\it HMHA1}, {\it MYO1G}, {\it LST1} & 0 & 0 & \\
			Cytokines and cytokine signaling & 7 & 6 & {\it CXCL16}, {\it CSF2RA}, {\it IFNGR1}, {\it IL10RA} & 1 & 1 & {\it MAP2K7}\\
			Metabolism & 9 & 9 & {\it PFKFB2}, {\it SLC2A9}, {\it CECR1}, {\it ALOX5} & 0 & 0 & \\
			Fc receptor & 6 & 4 & {\it FCGR2B}, {\it FCGR2A}, {\it FGL2}, {\it PTPN6} & 0 & 0 & \\
			Cytoskeleton & 5 & 5 & {\it HCLS1}, {\it WAS}, {\it IQGAP2} & 1 & 1 & {\it DNAI2}\\
			(An)ion transporters and channels & 4 & 4 & {\it SLCO2B1}, {\it SLC11A1} & 1 & 1 & {\it SLC24A4}\\
			AKT pathway & 3 & 3 & {\it PIK3IP1}, {\it PKIB} & 0 & 0 & \\
			Endocytosis & 3 & 3 & {\it APPL2}, {\it NECAP2} & 0 & 0 & \\
			Apoptosis, cell cycle control, and proliferation & 4 & 4 & {\it TMBIM4}, {\it TNFRSF1B}, {\it OGFRL1} & 1 & 1 & {\it BCCIP}\\
			Signaling & 4 & 4 & {\it RGS10}, {\it MFNG}, {\it FHL2}, {\it PILRA} & 0 & 0 & \\
			Growth hormone signaling & 0 & 0 & & 1 & 1 & {\it GHR}\\
			Morphogenesis & 0 & 0 & & 1 & 1 & {\it HOXC4}\\
			Others & 7 & 6 & {\it CUGBP2}, {\it CYP2S1}, {\it VAV1}, {\it GGN} & 2 & 2 & {\it NSUN5}, {\it MRPL4}\\
			Unknown & 16 & 16 & {\it VMO1}, {\it MICALCL}, {\it MS4A6A} & 6 & 6 & {\it NHN1}, {\it BRWD1}\\
			Total & 125 & 118 & & 14 & 14 & \\
			\hline
		\end{tabular}
		\caption{DE genes and probes by category comparing high\hyp{}grade osteosarcoma patients with and without metastases within 5 years by genome\hyp{}wide expression profiling (cohort 1).}
		\label{tab4.1}
	\end{table}
\end{landscape}
%

\subsection{Macrophage\hyp{}associated genes are expressed by infiltrating he\-ma\-to\-poi\-e\-tic cells and not by tumor cells}
The most probable source of expression of the DE macrophage\hyp{}associated genes was infiltrating immune cells and
not osteosarcoma cells. To confirm this, we performed qPCR
of {\it CD14} and {\it HLA-DRA} on osteosarcoma cell lines ($n=19$) and biopsies ($n=45$, a subset of cohort 1). {\it CD14} and
{\it HLA-DRA} expression was variable in osteosarcoma biopsies,
but almost undetectable in cell lines. This indicates that
these macrophage\hyp{}associated genes were not expressed by
tumor cells but by infiltrating cells because only osteosarcoma
biopsies contain macrophage infiltrate, whereas RNA
from cell lines is exclusively from tumor cells (Figure~\ref{fig4.1}A,
Mann\hyp{}Whitney U test p-value $<0.0001$).
%
\begin{figure}[htbp]
	\centering
	\includegraphics[width=1.0\textwidth]{figs04/fig1rgb.pdf}	% pdf version also rgb
	\caption{Macrophage\hyp{}associated genes are not expressed by osteosarcoma tumor cells. {\it A}, qPCR of osteosarcoma cell lines and biopsies of {\it CD14} and {\it HLA-DRA} demonstrating lack of expression by osteosarcoma cells. Mann\hyp{}Whitney U test p-value $<0.0001$, $\ast\ast\ast$. {\it B}, High expression of macrophage associated genes was associated with a better metastasis\hyp{}free survival (cohort 1, Kaplan\hyp{}Meier curve, p-value obtained by Logrank test, patients with metastasis at diagnosis have an event at $t=0$. These patients are included, because patients who develop metastases later on may as well have micrometastases at the time of diagnosis). Metastasis\hyp{}free survival curves for {\it HLA-DRA}, {\it CLEC5A}, and {\it FCGR2A} can be found in Supplemental Figure S2 ({\it available online}~\cite{ch4additional}). {\it C}, Double immunohistochemical staining of CD163 with the hematopoietic cell marker CD45 was performed and analyzed using spectral imaging microscopy. The pseudo\hyp{}IF image (pseudo\hyp{}IF) shows CD163\hyp{}positive cells in red, CD45\hyp{}positive cells in green, and colocalization of both markers in orange. Lack of expression of CD163 and CD45 on surrounding tumor cells (dark blue) and some single positive CD45 cells can be noted.}
	\label{fig4.1}
\end{figure}
%
In addition, we performed
double IHC for the hematopoietic cell marker
CD45, which is not expressed by osteosarcoma tumor cells,
and the macrophage marker CD163 or the macrophage\hyp{}associated
protein HLA-DR$\upalpha$ (Figure~\ref{fig4.1}C). We chose this
approach because no reliable osteosarcoma markers are
available~\cite{raymond2002conventional}. Our results confirmed that infiltrating hematopoietic
cells were the source of the macrophage\hyp{}associated
gene expression levels. Together, these data show that
osteosarcoma tumor cells do not express macrophage\hyp{}associated
genes, neither {\it in vitro} nor {\it in vivo}.

\subsection{Macrophage numbers in osteosarcoma biopsies
correlate with {\it CD14} gene expression levels and are
positively associated with localized disease and better
outcome (cohorts 2 and 3)}
To confirm the presence of TAMs in osteosarcoma, we
stained a tissue microarray containing 145 samples of 88
patients for the macrophage marker CD14 and counted the
number of positive cells per tissue microarray core (cohort
2; Figure~\ref{fig4.2}A).
%
%
\begin{figure}[htbp]
	\centering
	\includegraphics[width=1.0\textwidth]{figs04/fig2bw.pdf}	% OBS! print version bw
%	\includegraphics[width=1.0\textwidth]{figs04/fig2rgb.pdf}	% OBS! pdf version rgb
	\caption{{\it A}, Example of representative stainings of high\hyp{}grade osteosarcoma with high (left) versus low (right) levels of macrophage infiltration (CD14 staining) and vascular density (CD31 staining). {\it B}, High numbers of infiltrating macrophages (left, defined as the 3 upper quartiles, or more than 12 CD14\hyp{}positive cells per tissue array core) are associated with better OS (right, Logrank test p-value $=0.02$, cohort 2). Q1: lowest quartile, Q2, 3, 4, 3: highest quartiles.}
	\label{fig4.2}
\end{figure}
%
CD14 was chosen as opposed to CD68 because
the latter marker is not expressed by monocytes and often
shows cross\hyp{}reactivity with mesenchymal tissue ({\it data not
shown}). Number of CD14\hyp{}positive cells per tissue microarray
core correlated significantly with {\it CD14} mRNA expression
levels (14 samples overlap with gene expression
analysis, Spearman correlation coefficient 0.64, p-value $=0.01$). Similar to the gene expression data, there was a
trend for patients with primary localized disease to have
higher numbers of macrophages in pretreatment diagnostic
biopsies than patients with metastatic disease at presentation
(mean number of macrophages per core, 55 {\it vs} 27;
Mann\hyp{}Whitney U test p-value $=0.09$). Also, patients with
high macrophage counts at diagnosis tended to be less
likely to develop metastases within 5 years ($\chi^2$, p-value $=0.13$).

We subdivided this cohort into four quartiles based on
numbers of CD14\hyp{}positive cells to determine the group
with the best OS. No significant differences were found
between quartiles 2 and 4, but patients belonging to this
group had better OS than patients with low CD14 counts
(lowest quartile, or less than 12 CD14\hyp{}positive cells per
tissue array core; Figure~\ref{fig4.2}B, Logrank test p-value $=0.02$). In another
cohort of 16 patients, IF staining of CD14, CD163, and
HLA-DR$\upalpha$ was performed, again confirming a potential
prognostic value of high macrophage numbers (cohort 3,
Figure~\ref{fig4.3}, Logrank test p-value $=0.01$, Supplemental Figure S3 ({\it available online}~\cite{ch4additional})).
%
\begin{figure}[htbp]
	\centering
	\includegraphics[width=1.0\textwidth]{figs04/fig3rgb.pdf}	% pdf version also rgb
	\caption{{\it A}, Osteosarcoma samples are infiltrated with CD14 and CD163 single and double positive macrophages. Spectral imaging was used to reduce autofluorescence of osteosarcoma cells. In the composite image, CD14\hyp{}positive cells are represented in green, CD163\hyp{}positive cells are represented in red, and CD14/CD163 double positive cells are represented in yellow. Background autofluorescence of tumor cells is represented in gray. {\it B}, In an independent cohort of 16 patients (cohort 3), high macrophage infiltration as determined by IF CD14 staining was associated with significantly improved OS. p-values obtained using Logrank test, cutoff at the median.}
	\label{fig4.3}
\end{figure}
%

\subsection{Macrophages in osteosarcoma have both M1 and M2
characteristics}
To determine the phenotype of macrophages present in
osteosarcoma, we performed double IHC with CD14 and
either the M1\hyp{}associated marker HLA-DR$\upalpha$ or the M2\hyp{}associated
marker CD163. Not all CD163 and HLA-DR\hyp{}positive
infiltrating cells expressed CD14 (Figure~\ref{fig4.3}A and Supplementary
Figure S3A). The total number of macrophages as determined
by quantifying CD14\hyp{}positive macrophages was
associated with good survival (Figure~\ref{fig4.3}B), but the phenotype
of the macrophages (CD14/CD163 double positive versus
CD14/HLA-DR$\upalpha$ double positive) was not (Supplemental
Figure S3B ({\it available online}~\cite{ch4additional}); {\it data not shown}). Another M2 characteristic is
support of angiogenesis. The number of CD14\hyp{}positive
macrophages correlated with the number of CD31\hyp{}positive
vessels (Figure~\ref{fig4.2}A and Figure~\ref{fig4.4}), but vascularity did not correlate
with prognosis ({\it data not shown}).
%
\begin{figure}[htbp]
  \centering
  \begin{minipage}[b]{0.50\linewidth}
    \includegraphics[width=1\textwidth]{figs04/fig4bw.pdf}	% pdf version also bw
  \end{minipage}
    \hfill
  \begin{minipage}[b]{0.46\linewidth}
    \caption{Macrophage infiltration as determined by CD14\hyp{}positive cell count correlated with vascularity as determined by CD31\hyp{}positive vessel count. Data of all osteosarcoma samples (pre- and posttreatment primary tumor and metastatic samples, cohort 2) are shown. Q1: lowest quartile, Q2, 3, 4: three highest quartiles. Kruskal-Wallis test p-value $<0.0001$. $\ast$, Dunn's posttest p-value $<0.05$, $\ast\ast\ast$, Dunn's posttest p-value $<0.001$.}
     \label{fig4.4}
     \end{minipage}
\end{figure}
%

\subsection{Macrophage numbers in diagnostic biopsies may
predict histological response to chemotherapy and
macrophage number increases following
chemotherapy treatment}
There was a trend for high macrophage count (highest three
quartiles or more than 12 CD14\hyp{}positive cells per tissue
array core) in prechemotherapy diagnostic biopsies of the
primary tumor to predict for good histological response to
neoadjuvant chemotherapy (defined as more than 90\%
nonvital tumor tissue upon final resection), since 46\% of
patients with high macrophage numbers and 18\% of
patients with low macrophage numbers had a good histological
response (cohort 2; $\chi^2=0.09$). The prognostic
benefit of macrophage counts in osteosarcoma was not
independent of histological response using Cox proportional
hazard analysis. Macrophage numbers were higher
in postchemotherapy resections of the primary tumor than
in prechemotherapy biopsies (Supplemental Figure S4 ({\it available online}~\cite{ch4additional})).
Moreover, gene expression analysis showed upregulation
of macrophage\hyp{}associated probes in postchemotherapy
resections ($n=4$) as compared with prechemotherapy
biopsies ($n=79$, {\it data not shown}).

\section{Discussion}\label{discussion4}
OS of high-grade osteosarcoma patients with resectable
metastatic disease is poor at about 20\%~\cite{buddingh2010prognostic}. Mechanisms
for the development of metastases in osteosarcoma are
elusive. To identify genes that play a role in this process,
we performed genome\hyp{}wide expression profiling on
prechemotherapy biopsies of osteosarcoma patients. We
compared patients who developed clinically detectable
metastases within 5 years with patients who did not
develop metastases within this time frame (cohort 1).
About 20\% of genes overexpressed in patients without
metastases were macrophage\hyp{}associated, whereas an additional
25\% of genes had other immunological functions
({\it e.g.} in phagocytosis, complement activation or cytokine
production and response) but could still be attributed to
macrophages (Table 1 and Supplemental Table S4 ({\it available online}~\cite{ch4additional})). Thus,
in total, almost half of the DE genes belonged to one specific
process, {\it i.e.} macrophage function. Macrophage\hyp{}associated
genes were expressed by infiltrating hematopoietic cells
and not by osteosarcoma tumor cells (Figure~\ref{fig4.1}), indicating
a possible role for macrophages in preventing metastasis in
osteosarcoma. To confirm these findings, we quantified
infiltrating macrophages in two additional cohorts (cohorts 2
and 3) and found an association with better OS in both
cohorts.

The antimetastatic effect of TAMs in osteosarcoma is
remarkable because TAMs support tumor growth in a
substantial number of other cancers, which are mostly
tumors of epithelial origin. For example, macrophages
are associated with the angiogenic switch in breast cancer~\cite{lin2006macrophages}. We find an association between macrophage infiltration
and higher microvessel density, which suggests that
the influx of macrophages may support certain aspects of
tumor growth in osteosarcoma as well. However, in the
case of osteosarcoma, direct or indirect antitumor activity
of macrophages apparently outweighs their possible
tumor\hyp{}supporting effects. Macrophages can alter their phenotype
from M2 to M1 and become the tumor's foe instead
of its friend, given the right circumstances~\cite{hagemann2008re,sinha2005reduction,buhtoiarov2006macrophages}. The
TAMs that were identified in this study in osteosarcoma
had both M1 and M2 characteristics. The expression of
CD163 and the association with angiogenesis are M2
characteristics~\cite{lin2006macrophages,ojalvo2009high}. Some of the DE genes, such as
{\it MSR1} and {\it MS4A6A} are specific for M2 macrophages {\it in vitro}~\cite{martinez2006transcriptional}. Others, such as the proinflammatory cytokine {\it IL1B},
are more indicative of an M1 phenotype~\cite{mosser2008exploring}. How macrophages
inhibit osteosarcoma metastasis and whether a
balance between M1- and M2-type functions is responsible
is unknown.

In a multivariate regression model, the survival benefit
of high TAM numbers was at least partly dependent on
histological response to chemotherapy. Chemotherapy
can cause `immunogenic cell death' of cancer cells,
resulting in the release of endogenous danger signals~\cite{zitvogel2008immunological,kono2008dying}. The binding of these dangerous signals to
pattern recognition receptors on macrophages can skew
polarization of M2- to M1-type TAMs. The interaction
between dying tumor cells and resident TAMs may facilitate
clearance or inhibit outgrowth of metastatic tumor
cells. However, patients with localized disease at diagnosis
tended to have a larger macrophage infiltrate than
patients with metastatic disease at diagnosis (mean number
of macrophages per core 55 {\it vs} 27). At this point,
patients have not undergone chemotherapeutic treatment
yet and an interaction between chemotherapy and macrophages
can therefore not be responsible for the antimetastatic
effect of macrophages. Perhaps, the antimetastatic
effect of TAMs in these patients is due to the constitutive
presence of macrophages with an M1 phenotype. Alternatively,
the presence of macrophages might be a
reflection of a microenvironment not conducive for
metastasis.

Although preliminary analysis of a clinical trial investigating
the effect of treatment with the macrophage
activating agent MTP yielded conflicting results, subsequent
analysis revealed that treatment with MTP
improved 6-year OS from 70\% to 78\% in a cohort of
patients with primary localized disease~\cite{meyers2008osteosarcoma,meyers2005osteosarcoma}. Similar
results were obtained in canine osteosarcoma~\cite{kurzman1995adjuvant}. MTP
is a synthetic derivative of muramyl dipeptide (MDP), a
common bacterial cell wall component. Muropeptides
bind to intracellular pattern recognition receptors of
the nucleotide binding and oligomerization domain
(NOD)-like receptor (NLR) family, expressed
by macrophages~\cite{geddes2009unleashing}. In our study, 5 genes associated
with NLR family signaling and the associated `inflammasome'
were highly expressed in pretreatment biopsies of
patients who do not develop metastases. The DE genes
{\it NLRP3}, {\it NAIP}, {\it NLRC4}, and {\it PYCARD} are components of
the inflammasome, {\it LYZ} is a lysozyme that processes
bacterial cell wall peptidoglycan into MDP, a ubiquitous
natural analogue of MTP, and {\it IL1B} is the downstream
effector cytokine of the inflammasome pathway. Further
research is needed to clarify whether only patients with
high numbers of TAMs benefit from MTP treatment, or
whether MTP treatment is effective regardless of macrophage
number or activation status pretreatment. Also, it is
unknown whether treatment with agents promoting
macrophage migration or with other macrophage activating
agents like toll\hyp{}like receptor ligands or IFNs has a
similar beneficial effect on outcome.

Previous genome\hyp{}wide expression profiling studies in
osteosarcoma focused on identifying genes that predict
histological response to neoadjuvant chemotherapy~\cite{ochi2004prediction,salas2009molecular,man2005expression,mintz2005expression}.
As a consequence, the importance of macrophages in
controlling metastases was not recognized. However, we
previously compared gene expression profiles of osteosarcoma
biopsies and cultured mesenchymal stem cells
and determined which genes are expressed by tumor
stroma and not by tumor cells~\cite{cleton2009profiling}. There is a considerable
overlap between the stromal genes identified in
our previous study and the macrophage\hyp{}associated
genes identified in the present study (including HLA
class II genes as the most prevalent DE group of genes
and the macrophage\hyp{}associated genes {\it MSR1}, {\it MS4A6A},
and {\it FCGR2A}).

In conclusion, we showed the presence and clinical
significance of TAMs in pretreatment samples of high\hyp{}grade
osteosarcoma. TAMs in osteosarcoma are a heterogeneous
cell population with both M1 antitumor and M2 protumor
characteristics. Although the exact mechanism by which
macrophages exert their antimetastatic functions is still
unknown, this study provides an important biological
rationale for the treatment of osteosarcoma patients with
macrophage activating agents.

%%% references

\begin{small}
\begin{singlespace}
\bibliographystyle{unsrtnatshort}		% sorted as referenced, was unsrtnat, but unsrtnatshort gives shorter output
\bibliography{biblio}
\end{singlespace}
\end{small}

%\end{document}