% Marieke Kuijjer
% 2013-02-15
% chapter 09

	%\documentclass[12pt,b5paper]{book}
	%\setcounter{secnumdepth}{0}
	%\setcounter{tocdepth}{1}
	%\usepackage[hidelinks]{hyperref}

%\begin{document}

%%% title page

\chapter{Concluding remarks and future perspectives}\label{ch9}
\thispagestyle{empty}				%%% to remove page number from first page of chapter, must be placed after calling the chapter

%\vfill

\newpage
%%% main document
%%% first paragraph should not be an indent, so i set noident{} to this paragraph
\noindent{High-grade osteosarcoma is a primary bone tumor with complex genetic alterations, for which targeted therapy is lacking. The aim of this thesis was to use high\hyp{}throughput molecular data analysis of high\hyp{}grade osteosarcoma specimens and model systems, in order to learn more on osteosarcomagenesis and to find possible ways to inhibit this process. {\bf Chapter~\ref{ch1}} and {\bf Chapter~\ref{ch2}} give an introduction and literature review on microarray data analysis of high\hyp{}grade osteosarcoma.}

In {\bf Chapter~\ref{ch3}} we provide a rationale for the use of model systems for osteosarcoma. It describes differential expression for the clinical parameters sex, tumor location (femur, humerus, fibula/tibia), response to preoperative chemotherapy (poor responders, or Huvos grade 1--2, versus good responders, or Huvos grade 3--4), and histological subtype (osteoblastic, chondroblastic, fibroblastic osteosarcoma). Importantly, as we describe in a previous study performed by our group~\cite{cleton2009profiling}, no significantly differentially expressed genes were detected between poor and good responders to preoperative chemotherapy, even though a substantial amount of tumor samples was analyzed (see also Chapter~\ref{ch2}). Several publications do report differences between poor and good responders, but used relatively small sample sizes, and did not apply correction for multiple testing. An analysis of gene expression profiles of the three described histological subtypes showed that these differed significantly. Sets of fibroblastic- and of chondroblastic osteosarcoma\hyp{}specific genes were determined, and were enriched in genes with a role in cellular growth and proliferation and in the chondroid extracellular matrix, respectively. Using nearest shrunken centroids classification, an expression signature consisting of 24 probes that could predict for histological subtype was generated. This profile was validated on an independent dataset of osteosarcoma and control samples. Interestingly, this prediction profile was able to classify histological subtypes of the primary tumor from which the tested osteosarcoma xenografts and cell lines were derived, even though such material often lacks extracellular matrix. This implicates that the mRNA expression profiles of these model systems are representative for the primary tumor, and favor the use of osteosarcoma xenografts and cell lines in studying osteosarcoma biology. This is of particular importance given the rarity of this tumor as well as the difficulties to obtain adequate tissue.

%
\section{Targets for treatment of high-grade osteosarcoma}\label{targets9}
Different comparative analyses of the various types of data that were available led to the discovery of a number of particular ways to target this tumor. In {\bf Chapter~\ref{ch4}}, we compared patients who developed metastases within five years with patients who did not develop metastases within this time frame. The list of significantly differentially expressed genes was enriched in macrophage\hyp{}associated genes expressed by infiltrating cells (approximately 50\% of all genes), which were all overexpressed in patients who did not develop metastases. Tumor\hyp{}associated macrophages (TAM) of both the M1- (antitumor) and M2 (protumor)\hyp{}type were quantified with IHC in additional cohorts. The total count of M1- and M2-type macrophages was significantly correlated with a better overall survival. This is in contrast with most epithelial tumor types, which often show a correlation between (M2-type) infiltrating macrophages and poor survival. However, mesenchymal tumor cells may not need the guidance of the infiltrate to metastasize, and tumor\hyp{}associated macrophages may have a more antitumorigenic role in osteosarcoma~\cite{cleton2012immunotherapy}. Moreover, macrophages are plastic cells, and it could be that M2-type macrophages polarize towards M1-type macrophages after chemotherapy due to the release of danger signals by the dying tumor cells. The results of Chapter~\ref{ch4} provide a rationale for adjuvant treatment of high\hyp{}grade osteosarcoma patients with macrophage\hyp{}activating and recruiting agents, such as liposomal muramyl tripeptide phosphatidylethanolamine (L-MTP-PE). This drug has been previously shown to increase overall survival in canine and in human osteosarcoma~\cite{cleton2012immunotherapy,kager2010review}, although interpretation of the results obtained from the latter study has been difficult, due to the 2x2 factorial design; standard adjuvant chemotherapy treatment plus L-MTP-PE and/or ifosfamide. This study showed that two additional drugs did not show significant interaction (p-value $=0.101$) and therefore the treatment arms were pooled. A significant difference was then found for overall survival, but not for event\hyp{}free survival (EFS). In an unpooled analysis, EFS for patients treated with L-MTP-PE and ifosfamide was significantly improved when compared with patients treated with ifosfamide alone, but EFS arms of patients with only the standard adjuvant chemotherapy and patients who received L-MTP-PE without ifosfamide were not significantly different~\cite{kager2010review}. Further testing of this drug in osteosarcoma is therefore necessary, and this will, in the near future, be initiated in a phase II study in patients with metastatic and/or relapsed osteosarcoma.

In addition to differences between high\hyp{}grade osteosarcoma tumors with different clinical features, we studied common gene expression changes between sets of tumors and control samples. In {\bf Chapter~\ref{ch5}}, mRNA expression in osteosarcoma cell lines was compared with expression in osteosarcoma progenitors. Global pathway analyses pointed to differences in mRNA expression of the IGF1R pathway. Specifically genes negatively regulating this pathway upstream the IGF1 receptor showed downregulation in osteosarcoma, of the highest degree ({\it i.e.} the highest negative fold changes in the dataset). We therefore hypothesized that this pathway can be inhibited at the receptor level, and that this may inhibit growth of these tumors. Osteosarcoma cell lines were treated with a dual kinase inhibitor OSI-906, which inhibits both the insulin receptor (IR) and IGF1R, as IR can take over downstream signaling in case IGF1R is blocked, thereby inducing resistance to single IGF1R targeting~\cite{fulzele2007disruption,garofalo2011efficacy}. Inhibition with OSI-906 resulted in an inhibition of proliferation of 3/4 osteosarcoma cells, and may therefore be a promising drug for treatment in addition to adjuvant chemotherapy. Other pathways with a role in bone development, namely canonical Wnt signaling~\cite{cai2010inactive} and TGF$\upbeta$/BMP signaling~\cite{mohseny2012activities}, have been reported to play a role in osteosarcomagenesis. Because of the role of IGF1R signaling in bone development and growth, it is not surprising that this pathway is deregulated in osteosarcoma. Notably, in a recent case--parent study, two SNPs in the growth hormone (GH)/IGF1 pathway (in {\it IGF2R} and {\it IGFALS}) were significantly associated with osteosarcoma incidence~\cite{musselman2012case}. Interestingly, one of the very few genes which were overexpressed in patients developing metastases within 5 years (Chapter~\ref{ch4}) was the growth hormone receptor (GHR), which was also frequently amplified (in 34\% of all samples). IGF1 synthesis is largely dependent on growth hormone signaling~\cite{khandwala2000effects}, and an association between osteosarcoma and height/growth has been reported. The specific roles of the GH/IGF1 axis in osteosarcoma tumor growth and metastasis remain to be elucidated. The osteosarcoma cell line panel shows a variable expression of {\it GHR}, and includes four cell lines with high expression of {\it GHR}. These cell lines could be utilized to further experimentally examine this pathway in osteosarcoma.

{\bf Chapter~\ref{ch6}} described Ser/Thr kinome profiling analysis of two osteosarcoma cell lines using a peptide microarray. Although it is not yet possible to directly infer what kinase caused differential phosphorylation of the identified peptides, by pathway analysis we detected hyperphosphorylation directly downstream of Akt, pointing to active PI3K/Akt signaling. We treated osteosarcoma cells with MK-2206, an Akt inhibitor, which inhibited proliferation of 2 out of 3 cell lines. Inhibition of the PI3K/Akt signaling pathway may therefore also be a possible target for treatment of these tumors.

The effects of IGF1R and Akt inhibitors on osteosarcoma cell proliferation should be studied further, in order to determine why these cells stop proliferating after treatment (this may for example be ascribed to induction of apoptosis). In addition, the drugs need to be tested in combination with chemotherapy, in order to check for synergy and also in order to rule out toxicity of a combined treatment, as targeted treatment using a single drug against these signal transduction pathways will probably not be able eliminate all osteosarcoma tumor cells, and patients may develop resistance to targeted treatment. In addition, if targeted treatments will be used to treat osteosarcoma patients, the genomic and mutational status of associated pathway players should be determined, because patients with downstream aberrations may be insensitive to treatment, as was shown in Chapter~\ref{ch6} for the 143B cell line, which is insensitive to Akt inhibitor MK-2206, most probably due to its oncogenic transformation of {\it KRAS}.

%
\section{Integrative analysis and genomic instability}\label{integrative9}
A main finding of this thesis regards genomic instability, which appears to be affected in all data types studied---the genome, transcriptome, and kinome. In {\bf Chapter~\ref{ch6}}, an integrative analysis shows that pathways with a role in genomic stability are enriched for overexpressed genes, as well as for hyperphosphorylation of peptides, implying that not only gene expression, but also kinase activity is deregulated in these pathways.

In addition to complementing two different data types as was performed in Chapter~\ref{ch6}, integrative analysis can also be performed by applying intersections of both data types. This is useful in the analysis of copy number and gene expression data, because differently from kinase activity and transcript expression, the influence of DNA copy number of a specific gene on the mRNA expression levels of that particular gene is more direct than the influence of kinase activity on gene expression, since kinases usually act quite far upstream of transcription factors in a specific pathway. Different methods for performing integrative analyses on gene expression and copy number data exist, as is described in {\bf Chapter~\ref{ch7}} of this thesis. We tested the performance of two of such methods---a nonpaired and a paired analysis---on the high\hyp{}grade osteosarcoma dataset. In the nonpaired analysis, genes showing significant differential expression as compared with the control samples were returned when present in recurrent regions with copy number alterations that contained a higher number of significantly differentially expressed than expected by chance. For the paired analysis, a new approach was developed in statistical language R. This method determined cooccurrence of copy number changes and significant differential expression. A comparison of both methods on osteosarcoma data illustrated that the paired analysis returned more genes with biological relevance over a larger number of regions, even though fewer samples were used for this analysis, since complete RNA expression--DNA copy number pairs were available for 29/32 cases. By using a conservative approach and by combining the results of different paired analyses, we identified 31 candidate osteosarcoma drivers with high frequency of occurrence and significant differences in expression. While most of these genes were not yet reported in osteosarcoma, more than two\hyp{}thirds of the genes have been described to play a role in cancer. A large number of our candidate genes had a role in cell cycle regulation, stressing the possible role of genomic instability in driving osteosarcoma progression. This was further evaluated by calculating genomic instability scores, which showed that higher genomic instability correlated with poorer metastasis\hyp{}free survival. In addition, a negative correlation between the total amount of copy number aberrations and metastasis\hyp{}free survival was detected. We determined correlation with metastasis\hyp{}free survival, and not overall survival, because of the limited follow\hyp{}up available. Metastasis\hyp{}free survival, however, highly correlates with overall survival, as only a small percentage of patients with resectable metastases survive.

Some issues can be raised with regard to the selection of the method we applied to identify candidate driver genes. For determining recurrent copy number aberrations, we used a cut-off for frequency. This will result in the detection of mostly broad events ({\it e.g.} the amplification of an entire chromosome arm). Focal events may be detected as well, but this method of analysis does not directly pinpoint specific targets (alterations with selective benefits) of recurrent copy number aberrations. Focal events are most often determined by the identification of the minimal common region of overlap of a copy number aberration, but this approach is prone to misidentification of the driver gene, especially when the recurrence frequency of the driver event is low~\cite{mermel2011gistic2}, which is one of the reasons why we did not take this approach. Even though broad events do have a higher prevalence in most cancer types (low\hyp{}level aberrations affecting a chromosome arm or an entire chromosome), determining focal alterations may have great power to identify important genes in cancer~\cite{beroukhim2010landscape}. Importantly to note is also that both types of events may have different biological consequences. A method exist which can distinguish broad and focal events from each other and from background ({\it i.e.} passenger) events (GISTIC, Beroukhim {\it et al}.~\cite{beroukhim2007assessing}). It would be therefore interesting to perform this method to the osteosarcoma data set, in order to also identify significant recurrent focal aberrations. Results obtained with this analysis could return less frequently occurring aberrations, which are specifically selected for in a subset of the tumors. Using our integrative method, which does not filter out passenger copy number alterations statistically (as is done in GISTIC), but which integrates copy number with expression data, we were able to identify those significantly differentially expressed genes of which a large part (at least 35\%) of all tumors could be explained by an underlying copy number aberration. This approach gives us more information on osteosarcomagenesis in general ({\it i.e.} genes are affected in a high percentage of osteosarcoma samples). In addition to our list of new candidate osteosarcoma drivers, determining significant focal copy number events could provide us with some additional possible targets for treatment of a subset of osteosarcoma patients.

Candidate drivers need to be validated in an experimental setting. For the detected amplified and overexpressed genes this can be done by shRNA studies, but the effects of deleted tumor suppressors are more challenging to validate, especially because affecting a single gene will probably not be sufficient to stop tumor growth in cells with large amounts of aberrations. Because the majority of osteosarcomas do not have a known benign or less malignant precursor lesion, it is difficult to study tumor evolution, and to discriminate between early and late events in tumorigenesis of osteosarcoma. The detection of early drivers is important---it will reveal the first steps a normal cell takes in order to become tumorigenic, and these findings may be used in for example diagnostics. Genomic instability appears to play a major role in osteosarcoma, and in at least 25\%~\cite{stephens2011massive}, but probably in total approximately 50\% of all high\hyp{}grade osteosarcomas, this can be explained by chromothripsis. However, how chromothripsis exactly occurs, and what happens in the other half of osteosarcomas is yet unknown. By using the right model systems, these mechanisms can be studied. This is for example ongoing in studies which make use of the injection of different passages of transformed MSCs in mice and zebrafish~\cite{mohseny2012osteosarcomamodels,mohseny2012osteosarcomazebrafish} upon which their tumorigenicity can be assessed. Furthermore, conditional transgenic mouse models are useful tools to follow osteosarcomagenesis from the normal cell to the fully malignant tumor.

Finally, in {\bf Chapter~\ref{ch8}}, loss\hyp{}of\hyp{}heterozygosity (LOH) calls were integrated with copy number calls and expression. This approach identified a high cooccurrence of LOH and copy number gains. Such regions may harbor mutated tumor suppressor genes. Because the detected LOH may have been a technical artifact, we validated the LOH of a subset of tumor suppressors by Sanger sequencing and fluorescence in situ hybridization (FISH). Sanger sequencing may not be sensitive enough to pick up LOH in regions of high amplification, but using FISH we confirmed regions of low level gains. In these regions, the LOH which was detected with both the SNP microarray and Sanger sequencing was probably not an artifact. Mutation analysis of a subset of genes did, however, not detect any recurrent mutations. In order to improve the accuracy of mapping regions of LOH in osteosarcoma, an analysis which includes paired tumor--control samples should be performed.

FISH analysis detected homozygous staining regions (HSRs) in samples with high level gains. Oncogenes may be present in HSRs, and could be associated with chromothripsis. It will therefore be of interest to perform FISH on these regions of high amplification, to validate whether these oncogenes are indeed highly amplified. Targeting a specific oncogene that is highly amplified could be beneficial in osteosarcoma, as the tumor may be addicted to such an oncogene. In osteosarcoma, screening with high\hyp{}throughput methods for such events may detect possible patient\hyp{}specific treatment options.

%
\section{Summary}\label{summary9}
In summary, by high\hyp{}throughput data analysis of pretreatment biopsies of a relatively large, homogeneous cohort of osteosarcoma patients, which was collected as a collaborative effort by EuroBoNeT, we discovered a protective role of macrophages against the development of metastases. In addition, the IR/IGF1R and PI3K/Akt signaling pathways were discovered as potential targets for treatment. By integrative genomic analyses, the genomic complexity of this tumor was confirmed, and a correlation of genomic complexity with metastasis\hyp{}free survival was identified. A conservative integrative approach to filter out passenger genes from driving events resulted in a list of mostly new, highly frequent candidate drivers in osteosarcoma. Most convincingly, genes playing a role in the maintenance of genomic stability have a considerable driving role in osteosarcomagenesis, as these pathways were affected in all three data types studied (mRNA expression, copy number data, and the kinome screen). Figure~\ref{fig9.1} summarizes the results obtained from these studies.
%
\begin{figure}[h] % OBS! changed [htbp]
	\centering
	\includegraphics[width=1.0\textwidth]{figs09/fig1bw.pdf}	% pdf version also bw
	\caption{Schematic overview of the results described in this thesis. {\it A}, Genomic instability and low macrophage count in the primary tumor are associated with poor prognosis and IGF1R and Akt signaling pathways are active in osteosarcoma. {\it B}, Low genomic instability scores are associated with better prognosis. Possible ways to intervene tumor progression are activation of macrophages with L-MTP-PE and inhibition of IR/IGF1R and Akt signaling with OSI-906 and MK-2206, respectively.}
	\label{fig9.1}
\end{figure}
%
This thesis provides the first steps in unraveling the genomic and transcriptomic landscape of this highly genomically unstable tumor. The research of osteosarcoma genomics at an even higher resolution (Next Generation Sequencing) will, together with the proposed future studies discussed in this chapter, help to better understand this highly genomically unstable tumor, and will provide indispensable knowledge on cancer evolution, diagnostics, prognostics, and targeted therapies.

%%% references

\begin{small}
\begin{singlespace}
\bibliographystyle{unsrtnatshort}		% sorted as referenced, was unsrtnat, but unsrtnatshort gives shorter output
\bibliography{biblio}
\end{singlespace}
\end{small}

%\end{document}